project/EnvironmentCheck.scala

```diff
@@ -105,8 +105,7 @@ object EnvironmentCheck {
       val newState = oldTransition(state)
       val logger   = newState.log
       val graalOk  = graalVersionOk(graalVersion, javaVersion, logger)
-      val rustOk   = rustVersionOk(rustVersion, logger)
-      if (!graalOk || !rustOk)
+      if (!graalOk)
```

Comment:
I think the idea here was to at least issue a warning on startup (without blocking the startup by throwing an error) if the Rust version doesn't match.

And a decision is made to not print a warning on startup, please remove the `rustVersion` argument from this function, as it would be unused.